### PR TITLE
UNI-528: Disable model creation on param finder failures

### PR DESCRIPTION
**Pull request by lscheinkman**
@marionleborgne Please review

CC: @cbaranski

### CODE REVIEW

File: unicorn/app/config/default.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
         "title": "HTM analysis for %s",
         "recommendAggregate": "We found the best parameters for the HTM ​and your data will be aggregated​. While you can tweak them in the advanced settings below, we recommend that you proceed with these parameters for the analysis.",
         "recommendRaw": "We found the best parameters for the HTM​. While you can tweak them in the advanced settings below, we recommend that you proceed with these parameters for the analysis.",
+        "paramFinderError": "We were unable to find parameters for this metric. Please check your data file.",
```
Comment (marionleborgne) [inline unicorn/app/config/default.json]: Just add a updated message:

```
We were unable to find parameters for this metric. Please contact support at htm-studio@numenta.com.
```

Comment (lscheinkman) [inline unicorn/app/config/default.json]: error message
Here is the new error message

**Comment by marionleborgne** [comment]
👍

File: unicorn/app/config/default.json (hunk 1 above)
Comment (marionleborgne) [inline unicorn/app/config/default.json]: Minor comment: missing dot at the end. You can merge after that 👍

Comment (marionleborgne) [inline unicorn/app/config/default.json]: Minor comment: missing dot at the end. You can merge after that 👍

**Pull request merged**